src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures.yml

```diff
@@ -92,25 +92,25 @@ sylius_fixtures:
                     options:
                         custom:
                             clothing:
-                                name: Clothing
+                                name: "Clothing"
                             books:
-                                name: Books
+                                name: "Books"
                             other:
-                                name: Other
+                                name: "Other"
 
                 tax_rate:
                     options:
                         custom:
                             clothing_tax:
-                                name: Clothing Sales Tax 7%
+                                name: "Clothing Sales Tax 7%"
                                 zone: US
                                 category: clothing
                             books_tax:
-                                name: Books Sales Tax 2%
+                                name: "Books Sales Tax 2%"
                                 zone: US
```

Comment:
Because it is a code of zone, not string itself. Should we also add quotation around them?